Lesson3/ConsoleAppExtensions.cs

```diff
@@ -0,0 +1,118 @@
+﻿using Project.Interface;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Project.ConsoleApp
+{
+    public static class ConsoleAppExtensions
+    {
+        const string _json = "-json";
```

Comment:
`Observation`: If these were public then they would be usable from the test projects which would keep the changes in sync.